packages/card/src/Card.ts

```diff
@@ -149,9 +169,35 @@ export class Card extends ObserveSlotPresence(
         }
     }
 
+    private stopPropagationOnHref(event: Event): void {
+        if (this.href) {
+            event.stopPropagation();
+        }
+    }
+
+    private handlePointerdown(event: Event): void {
```

Comment:
For my curiosity - how did we get to timing pointer down/up ourselves instead of using click events? Is it not possible to listen for clicks while ignoring clicks that originated on links in the content?

Reply:
While this also allows us to avoid links in the content, the timing here is actually to allow for users to select the content in the Card element, as well as click the card to interact with it, as per: https://inclusive-components.design/cards/#theredundantclickevent

Reply:
💡  nice